frontend/src/utils/storage.ts

```diff
@@ -29,6 +29,12 @@ export const saveJobToCache = (
   storage.setItem(jobCacheKey, JSON.stringify(newCachedJobEntry));
 };
 
+export const clearJobFromCache = (jobId: string) => {
```

Comment:
This should never be needed except when restarting a development chain and reusing job IDs.  This should be changed to IPFS CID instead of job id, because all it is doing is caching IPFS metadata (which is immutable).

It is fine for now.  I added to the issue.